plugins/core/gov.nasa.jpl.imce.oml.dsl/src/gov/nasa/jpl/imce/oml/dsl/OML.xtext

```diff
@@ -540,7 +540,7 @@ terminal IRI:
 	'<' (!'>')* '>';
 
 terminal ABBREV_IRI:
-	('a'..'z'|'A'..'Z'|'_'|'0'..'9')('a'..'z'|'A'..'Z'|'_'|'-'|'.'|'0'..'9')+ ':' ('a'..'z'|'A'..'Z'|'_'|'0'..'9')('a'..'z'|'A'..'Z'|'_'|'-'|'.'|'0'..'9')+;
+	('a'..'z'|'A'..'Z'|'_'|'0'..'9')('a'..'z'|'A'..'Z'|'_'|'-'|'.'|'0'..'9')* ':' ('a'..'z'|'A'..'Z'|'_'|'0'..'9')('a'..'z'|'A'..'Z'|'_'|'-'|'.'|'0'..'9')*;
```

Comment:
The grammar was too restrictive; it required a minimum of 2 characters before and after ':'.
The updated rule requires a minimum of 1 character before and after ':'